src/block_cache/block_cache.c

```diff
@@ -871,6 +871,15 @@ __wt_blkcache_setup(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
     if (cval.val == 0)
         return (0);
 
+    /* FIXME-WT-15663 Disable block cache until it is stable */
+    if (cval.val == 1) {
+        __wt_verbose_warning(session, WT_VERB_BLKCACHE,
+          "Block cache is currently disabled. Continuing without block cache "
+          "enable:%" PRIi64,
+          cval.val);
```

Comment:
I don't think we need to print `cval.val` here.
```suggestion
        __wt_verbose_warning(session, WT_VERB_BLKCACHE,
          "Block cache is currently disabled. Continuing without block cache "
          "enabled.");
```

Reply:
It's simply because function `__wt_verbose...` always need at least one param to the format string, otherwise there will be a compile error `expect an expression`

Reply:
You can do:
```
        __wt_verbose_warning(session, WT_VERB_BLKCACHE, "%s",
          "Block cache is currently disabled. Continuing without block cache enabled.");
```

Reply:
Oh, better solution for this, TKS!